katas/content/nonlocal_games/index.md

```diff
@@ -53,9 +51,147 @@ To start with, let's take a look at how you would play the classical variant of
     "path": "./chsh_classical_strategy/"
 })
 
+@[exercise]({
+    "id": "nonlocal_games__chsh_quantum_alice_strategy",
+    "title": "Alice's Quantum Strategy",
+    "path": "./chsh_quantum_alice_strategy/"
+})
+
+@[exercise]({
+    "id": "nonlocal_games__chsh_quantum_bob_strategy",
+    "title": "Bob's Quantum Strategy",
+    "path": "./chsh_quantum_bob_strategy/"
+})
+
 @[section]({
-    "id": "nonlocal_games__conclusion", 
-    "title": "Conclusion" 
+    "id": "nonlocal_games__discussion",
+    "title": "Discussion: Probability of Victory for Quantum Strategy"
+})
+
+The above quantum strategy adopted by Alice and Bob offers a win rate of $\frac{2 + \sqrt{2}}{4}$, or about 85.36%. Let's see why this is the case.
```

Comment:
The formatting will look a bit nicer if we render the probabilities as formulas; this will make them stand out from the text, same as the numbers that lead to them do later in the discussion

Reply:
This and other probabilities (in current kata) were change to formula-based.
I'll follow this approach for other games, where probability have to be stand out.